.github/workflows/ci.yml

```diff
@@ -19,12 +19,37 @@ jobs:
         go version
         go install golang.org/x/lint/golint
 
-    - name: Run go fmt & lint
+    - name: Run Go fmt
       run: |
-        go fmt -mod=readonly ./...
-        git diff HEAD
-        if [ `git diff HEAD --name-only | wc -l` -gt 0 ]; then exit 1; fi
-        golint -set_exit_status=1 `go list -mod=readonly ./... | grep -v /vendor/`
+        unformatted=$(IS_CONTAINER=FALSE ./hack/go-fmt.sh .)
```

Comment:
i'm not sure, but i'm guessing that the issue you saw might be worked around by putting quotes around this expression or something to that effect. did you try any other ways to format this?